AUTHORS

```diff
@@ -2236,5 +2236,8 @@ Shelley Vohr <shelley.vohr@gmail.com>
 Deepjyoti Mondal <djmdeveloper060796@gmail.com>
 Brett Kiefer <brett@trello.com>
 Kevin Thomas <kevintab95@gmail.com>
+Richard Chamberlain <richard_chamberlain@uk.ibm.com>
+Manusaporn Treerungroj <m.treerungroj@gmail.com>
+Julian Alimin <unknown>
```

Comment:
We really should have an email address for this one.

Reply:
The AUTHORS file should not be updated manually in this PR. We have `tools/update-authors.sh` for that and usually do it with a standalone PR.

Reply:
So that's an interesting question. These are authors for node-report who do not currently have commits in core so presumably won't be picked up by the tooling.

Reply:
I think what we are supposed to do to give attribution is to update the license with https://github.com/nodejs/node-report/blob/master/LICENSE.md.

Reply:
As far as attribution is concerned, there’s the `Co-authored-by:` field that is now recognized by Github. We should probably  add support for it to our own `AUTHORS` tooling?

Reply:
done, all the authors of the `node-report` module is now sourced into the commit message as co-authors.